.github/workflows/compile.yml

```diff
@@ -51,12 +51,12 @@ jobs:
       - uses: actions/upload-artifact@v4
         with:
           path: ./build/libllama.so
-          name: llama-bin-linux-${{ matrix.build }}-x64.so
+          name: llama-bin-linux-${{ matrix.build }}-x64
```

Comment:
@martindevans Could you please take a look of this? I'm not sure what it's expected to do with the original code.